Scala/Scala.sublime-syntax

```diff
@@ -147,6 +147,39 @@ contexts:
     - match: '{{typeprefix}}'
       push: single-type-expression
 
+  annotation:
+    - match: '@{{plainid}}\.'
```

Comment:
Yes, that is correct, the `@` should be in addition to the `storage.modifier.annotation.scala` scope.